authors/apps/comments/views.py

```diff
@@ -313,7 +315,11 @@ def get(self, request, comment_pk, pk):
                 "status": status.HTTP_404_NOT_FOUND
             }, status=status.HTTP_404_NOT_FOUND)
         comment_reply = CommentReply.objects.get(pk=pk)
-        self.check_object_permissions(request, comment_reply)
+        if comment_reply.author.user != request.user:
```

Comment:
@FahdJamy, good work here. May I please understand why you had to take this validation out of the permission class?

Reply:
I was asked to write a custom returned message so I had to deal away with that.
Thanks for that though.